tests/auto_launch/test_client_auto_launch.py

```
@@ -0,0 +1,98 @@
+    # Should have the correct configuration
+    assert client.config.url == "http://127.0.0.1:7700"
+    assert client.config.api_key == "masterKey"
+
+
+def test_client_del_cleanup():
+    """Test that __del__ properly cleans up the local server."""
+    client = meilisearch.Client(api_key="cleanup_test_key")
+    assert client._local_server is not None
+
+    # Store reference to local server
+    local_server = client._local_server
```

Ruff (0.11.9)

Local variable `local_server` is assigned to but never used

Remove assignment to unused variable `local_server`

(F841)